perceiver/model/lightning.py

```diff
@@ -74,6 +74,10 @@ def test_step(self, batch, batch_idx):
 
 
 class LitImageClassifier(LitClassifier):
+    """
+    >>> lit = LitClassifier((64, 64, 3), 2)
+    """
+
```

Comment:
Why has this been added?

Reply:
because your project does not have any test, so pytest fails at this moment as there is nothing to test...
and this is very minimal kind of smoke test :rabbit: